services/web/server/tests/unit/with_dbs/02/test_projects_crud_handlers__list_with_query_params.py

```diff
@@ -360,3 +364,106 @@ async def test_list_projects_with_order_by_parameter(
     assert [
         item["description"][0] for item in data["data"]
     ] == _alphabetically_ordered_list
+
+
+@pytest.fixture()
+def setup_folders_db(
+    postgres_db: sa.engine.Engine,
+    logged_user: UserInfoDict,
+    user_project: ProjectDict,
+) -> Iterator[FolderID]:
+    with postgres_db.connect() as con:
+        output = []
+        result = con.execute(
+            folders.insert()
+            .values(
+                name="My Folder 1",
+                description="My Folder Decription",
+                product_name="osparc",
+                created_by=logged_user["primary_gid"],
+            )
```

Comment:
Please use the faker for the name and description.
for the product_name there is a fixture called product_name see [https://github.com/ITISFoundation/osparc-simcore/blob/ad25d00d8c12c91849932ba0c1920c616d36f182/packages/pytest-simcore/src/pytest_simcore/faker_products_data.py](https://github.com/ITISFoundation/osparc-simcore/blob/ad25d00d8c12c91849932ba0c1920c616d36f182/packages/pytest-simcore/src/pytest_simcore/faker_products_data.py)see if that can be re-used.